test/test_browser.py

```diff
@@ -5500,6 +5500,13 @@ def test_webpack(self, es6):
     shutil.copy('webpack/src/hello.wasm', 'webpack/dist/')
     self.run_browser('webpack/dist/index.html', '/report_result?exit:0')
 
+  def test_vite(self):
+    shutil.copytree(test_file('vite'), 'vite')
+    with common.chdir('vite'):
+      self.compile_btest('hello_world.c', ['-sEXPORT_ES6', '-sEXIT_RUNTIME', '-sMODULARIZE', '-o', 'hello.mjs'])
```

Comment:
Do we want to use `MODULARIZE=instance` here since that's probably more similar to what vite users are using?

Reply:
This is just the first of (hopefully many) different tests.  I'm not sure I'd want to use `-sMODULARIZE=instance` for this initial smoketest since its still experimental.

Reply:
The next test I plan to add is one that uses threads so we can confirm that #22394 really is fixed.